package.json

```diff
@@ -83,5 +89,9 @@
     "webpack-cli": "^3.3.12",
     "webpack-dev-server": "^3.11.0"
   },
-  "dependencies": {}
+  "dependencies": {
+    "carbon-components": "^10.20.0",
```

Comment:
Please place `carbon-components` and `carbon-icons` in `devDependencies` and mark them as peers (i.e., place them in `peerDependencies`, too).

Also introduce them as shared variables - they are already part of the app shell.